cantabular/static_dataset_test.go

```diff
@@ -773,6 +809,99 @@ var expectedDimensions = cantabular.GetDimensionsResponse{
 	},
 }
 
+// mockRespBodyGetGeographyDimensions is a successful 'get geography dimensions' query respose that is returned from a mocked client for testing
+var mockRespBodyGetGeographyDimensions = `
+{
+	"data": {
+		"dataset": {
+			"ruleBase": {
+				"isSourceOf": {
+					"edges": [
+						{
+							"node": {
+								"categories": {
+									"totalCount": 2
+								},
+								"label": "Country",
+								"mapFrom": [
+									{
+										"edges": [
+											{
+												"node": {
+													"filterOnly": "false",
+													"label": "Region",
+													"name": "Region"
+												}
+											}
+										]
+									}
+								],
+								"name": "Country"
+							}
+						},
+						{
+							"node": {
+								"categories": {
+									"totalCount": 10
+								},
+								"label": "Region",
+								"mapFrom": [],
+								"name": "Region"
+							}
+						}
+					]
+				},
+				"name": "Region"
+			}
+		}
+	}
+}
+`
+
+var expectedGeographyDimensions = cantabular.GetGeographyDimensionsResponse{
```

Comment:
as far as I can see, we're only round-tripping in the test - do we need all this detail?

Reply:
I used real responses for the tests, maybe I can move them to a separate file, so that the testing code is cleaner?

Reply:
dunno - if that's the way we generally document expected responses etc. that's fine. Could be cleaner in a new file maybe but meh...